runtime/lib/ttnn/operations/creation/empty.cpp

```diff
@@ -18,8 +18,8 @@ void run(const ::tt::target::ttnn::EmptyOp *op, ProgramContext &context) {
   ::ttnn::Layout layout __attribute__((unused)) =
       ::tt::runtime::ttnn::utils::toTTNNLayout(op->layout());
   layout = ::ttnn::Layout::ROW_MAJOR;
-  ::ttnn::Shape shape = ::ttnn::Shape(
-      tt_metal::Shape(::tt::runtime::ttnn::utils::toShapeFromFBShape(
+  ::ttnn::Shape shape = ::ttnn::Shape(::tt::tt_metal::LegacyShape(
```

Comment:
I assume this is a workaround, but can we create (if not already) an issue to utilize newly defined `Shape`, instead of relaying on Legacy one? 

cc @sdjordjevicTT

Reply:
@nvukobratTT this isn't a workaround, we're creating a `::ttnn::Shape` (I assume this is what you mean by newly defined Shape) which is a wrapper around `tt_metal::LegacyShape`.